Week3/homework/step3.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+function createBase(a) {
+  return function(y) {
+    console.log(y + a);
```

Comment:
The function should be responsible for returning a number, not console logging. You should return y + a here instead of console logging it.